openedx/features/caliper_tracking/utils.py

```diff
@@ -25,3 +25,14 @@ def convert_datetime(current_datetime):
         utc_datetime.strftime(UTC_DATETIME_FORMAT)[:-3], 'Z'
     )
     return formatted_datetime
+
+
+def get_username_from_user_id(user_id):
+    """
+    @param : user_id
+    :return: username from the given user_id.
+    """
+
+    from django.contrib.auth.models import User
```

Comment:
Please use the `get_user_model()` function to get the user model at run time instead of directly importing. 
 
https://docs.djangoproject.com/en/1.11/topics/auth/customizing/#django.contrib.auth.get_user_model

Reply:
done